### DISCUSSION TITLE
Availability message live maps for mappers

### DISCUSSION THREAD

**Discussion by czanongit** [osmandapp/OsmAnd#15433] [General]
### Description
I consider that the message stating the duration of live maps for mappers using OSM editor plug-in is inaccurate. Maps are not available until 16th day of the month after last two with sufficient edits. Live mode is available until the start of next month, because the computation consider the current plus the previous month edits. So an entire month give always the same result. 

### How to reproduce?
Using OSM editor Plug in, with an OSM profile that has the correct edits count, than you need to wait next month. See attachments

### Your Environment
OsmAnd Version: OsmAnd 4.2.7, rilasciata: 2022-07-13
Android/iOS version: Android 12
Device model: not relevant 

**Maps used (online or offline):**   
Not related to offline maps. 

![IMG_20221004_235925_253](https://user-images.githubusercontent.com/91224850/193938189-18cc4bac-ef76-48c6-8a54-7925d5eb2c15.jpg)

![IMG_20221004_235927_880](https://user-images.githubusercontent.com/91224850/193938233-ae0c8a13-1ab2-4ca8-bdb7-be8ebb71c9f0.jpg)

**Comment by vshcherb** [comment]
We did intentionally 15 changes per month but because at the beginning of the month you have 0, we sum up previous month as well. So to stay ok, you need to edit 30 objects each month.

**Reply by czanongit** [reply]
Ok, right, this sounds very good.
But the explaining message in the app states differently: I would prefer to read the following in order to understand what you are saying:
<<a minimum of 15 changes every month is required to stay active, or at least from the start of the month before the current>>
 and similarly in the title I think one of these messages could explain more precisely the case:
- < < maps available until the end of next month > > when you have sufficient changes during the current month;
- < < maps available until the end of this month > > when you have sufficient changes during the last month and not this one.
